src/landing/home-stats/home-stats-table.jsx

```diff
@@ -267,8 +283,10 @@ function getLabelForStatKey(key) {
  */
 function HandleLinkRenderer(params) {
     try{
-      return (<a href={new URL(params.data.ProfileURL).href} target="_blank">{params.data.Handle}</a>);
+      return (<Link to={`/${unwrapShortHandle(params.data.handle)}/history`} className={'account-short-entry'}>
+        {params.data.handle}
+        </Link>)
     }catch(error){
-      return params.data.Handle
+      return params.data.handle
     }
```

Comment:
with the removal of the handle from the api return this whole component will need to use a lookup hook to get the handle for display. it will be best to reuse an existing component for this purpose, like AccountShortEntry (at `src/common-components/account-short-entry.jsx`)